tests/cloud_test.py

```diff
@@ -261,17 +262,18 @@ async def test_meijucloud_download_lua(self) -> None:
         assert cloud is not None
         assert await cloud.login()
 
-        if not os.path.exists("/tmp/download"):
-            os.mkdir("/tmp/download")
-        file = await cloud.download_lua("/tmp/download", 10, "00000000", "0xAC", "0010")
+        tmp_download = os.path.join(gettempdir(), "download")
+        if not os.path.exists(tmp_download):
```

Comment:
define the path as a variable so to reuse it in both places and assure it's consistent